application/lib/tutorial/two/module/explore_menu.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/semantics.dart';
+
+class ExploreMenuPage extends StatefulWidget {
+  const ExploreMenuPage({super.key});
+
+  @override
+  State<ExploreMenuPage> createState() => _ExploreMenuPageState();
+}
+
+class _ExploreMenuPageState extends State<ExploreMenuPage> {
+  bool _hasSpokenIntro = false; // Whether the intro has been spoken yet
+
+  void _speakIntro() {
+    SemanticsService.announce(
```

Comment:
Did this really work on iOS? Might need to get @MartyCarroll to check on physical iPhone, since we had issues with `SemanticsService` being overridden by VoiceOver speaking out the currently focused widget.